### PR TITLE
[Updates] Cloud manager updates

### PR DESCRIPTION
**Pull request by leslitagordita**
* Misc. updates to cloud manager docs.

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit c3a2e9df7facdff56afad551cc60dfbdb715b7ba

https://deploy-preview-3035--nostalgic-ptolemy-b01ab8.netlify.com

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit e724fbe8a3a9a1c2c44dda5630d19f803553f136

https://deploy-preview-3035--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/security/linode-manager-security-controls/index.md [hunk 1]
```diff
@@ -150,6 +150,21 @@ Should you need us to disable your Two-Factor Authentication, the following info
 1. An image of the front and back of the payment card currently associated with your account, which clearly shows the last 6 digits, expiration date, and cardholder name.
 1. An image of the front and back of a matching government-issued photo ID.
 
+## Manage Trusted Devices
+
+Whenever you log into the Cloud Manager from a new device or computer, you can choose to _Trust this computer for 30 days_. This means device will not be prompted for a username or password for 30 days. The Cloud Manager displays a list of all trusted devices for your account and gives you the ability to *untrust* any device on the list. Your account's associated email address will also be notified whenever a new device is trusted. To view and manage your account's trusted devices:
```
Comment (andystevensname) [inline docs/security/linode-manager-security-controls/index.md]: This means _your_ device

File: docs/security/linode-manager-security-controls/index.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ Enable two-factor authentication to start using it with your Linode account.
 1.  In the **Two-Factor Authentication (TFA)** section, toggle the **Disabled** switch so that it reads **Enabled** to enable Two-Factor Authentication.
 1.  A new form (depicted below) will appear. Write down the **Secret Key** and store it in a safe place:
 
-    [![The key and QR code for two-factor authentication.](security-tfa-settings-small.png "The key and QR code for two-factor authentication")](security-tfa-settings.png)
+    ![The key and QR code for two-factor authentication.](security-tfa-settings-small.png "The key and QR code for two-factor authentication")](security-tfa-settings.png)
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/security/linode-manager-security-controls/index.md]: Fixed!

File: docs/security/linode-manager-security-controls/index.md [hunk 3]
```diff
@@ -117,12 +117,12 @@ The Linode Cloud Manager allows you to generate a new secret key for your two-fa
...
 1.  Select the **My Profile** link by clicking on your username at the top of the page:
 
-   ![Select 'My Profile'](security-my-profile.png "Select 'My Profile')
+   ![Select 'My Profile'](security-my-profile.png "Select My Profile")
 
 1.  Select the **Password & Authentication** tab.
 1.  In the **Two-Factor Authentication (TFA)** section, click **Reset two-factor authentication**, as shown below.
 
-    [![Reset two-factor authentication.](security-reset-tfa-small.png "Reset two-factor authentication.")](security-reset-tfa.png)
+    ![Reset two-factor authentication.](security-reset-tfa-small.png "Reset two-factor authentication.")](security-reset-tfa.png)
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/security/linode-manager-security-controls/index.md]: Fixed!

File: docs/security/linode-manager-security-controls/index.md (hunk 1 above)
Comment (leslitagordita) [reply docs/security/linode-manager-security-controls/index.md]: Fixed!

**Pull request merged**